apps/payments/next/.env.development

```diff
@@ -51,6 +51,9 @@ CONTENTFUL_CLIENT_CONFIG__FIRESTORE_CACHE_COLLECTION_NAME=contentfulQueryCacheCo
 CONTENTFUL_CLIENT_CONFIG__MEM_CACHE_T_T_L=
 CONTENTFUL_CLIENT_CONFIG__FIRESTORE_CACHE_T_T_L=
 
+# Contentful Service Config
+CONTENTFUL_SERVICE_CONFIG__SUPPORTED_PLAN_IDS=plan_GqM9N6qyhvxaVk,price_1KbomlBVqmGyQTMaa0Tq7UaW,price_1Ivq4gBVqmGyQTMaplHcFEGO
```

Comment:
Do these get used somewhere? Only reference to the env vars I'm seeing are this and in .env.production#L49 below, but nobody reading from that env var (or config file referencing that config):
```sh
git grep "CONTENTFUL_SERVICE_CONFIG__SUPPORTED_PLAN_IDS" | cat

apps/payments/next/.env.development:CONTENTFUL_SERVICE_CONFIG__SUPPORTED_PLAN_IDS=plan_GqM9N6qyhvxaVk,price_1KbomlBVqmGyQTMaa0Tq7UaW,price_1Ivq4gBVqmGyQTMaplHcFEGO
apps/payments/next/.env.production:CONTENTFUL_SERVICE_CONFIG__SUPPORTED_PLAN_IDS=
```